docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -1025,7 +1025,7 @@ Here is the order of precedence from least to greatest (the last listed variable
   #. task vars (only for the task)
   #. include_vars
   #. set_facts / registered vars
-  #. role (and include_role) params
+  #. role params
```

Comment:
I'd avoid removing that until there really is no such thing as the behavior of role_params, or otherwise list it as:
```suggestion
  #. role params (and task vars on include_role or import_role)
```

Reply:
`vars: ` is different precedence than params. There is ongoing discussion about this on IRC right now.